lintro/tools/core/tool_registry.py

```
@@ -431,3 +442,32 @@ def __len__(self) -> int:
     def __contains__(self, name: str) -> bool:
         """Check if a tool is in the registry."""
         return name in self._tools
+
+
+def __getattr__(name: str) -> Any:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)